app/src/main/res/layout-land/onboarding_fragment.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="presenter"
+      type="org.oppia.app.onboarding.OnboardingFragmentPresenter" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.onboarding.OnboardingViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/oppiaBackgroundYellowIvory">
+
+    <androidx.viewpager.widget.ViewPager
+      android:id="@+id/onboarding_slide_view_pager"
+      android:layout_width="match_parent"
+      android:layout_height="0dp"
+      android:layout_marginBottom="0dp"
+      android:overScrollMode="never"
+      android:scrollbars="none"
+      app:layout_constraintBottom_toTopOf="@id/bottom_container"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent">
```

Comment:
We can make this a self-closing tag 
`app:layout_constraintTop_toTopOf="parent" />`

Reply:
Done 👍